enterprise/views.py

```diff
@@ -468,6 +478,26 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
 
         """
         try:
+            enterprise_uuid = UUID(enterprise_uuid)
+            # pylint: disable=no-member
+            enterprise_customer = EnterpriseCustomer.objects.get(uuid=enterprise_uuid)
+        except (ValueError, EnterpriseCustomer.DoesNotExist):
+            logger.error('Unable to find enterprise customer for UUID: %s', enterprise_uuid)
+            raise Http404
+
+        # Now verify if the user is logged in. If user is not logged in then
+        # send the user to the login screen to sign in with an
+        # Enterprise-linked SSO and the pipeline will get them back here.
+        if not request.user.is_authenticated():
+            return redirect(
+                '{login_url}?next={current_url}?{query_string}'.format(
+                    login_url=reverse('signin_user'),
```

Comment:
Don't use reverse here. We may someday want edx-enterprise to run as a separate IDA.